Dev-Corner/slides/conda.html

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
I didn't mean to include this file in this commit - it is WIP. Opps - I'll rework.